metal_small/data/cut_data.py

```diff
@@ -0,0 +1,29 @@
+from numpy.random import default_rng
+
+from astropy.table import Table
+
+
+def sample_obsdata(filename, nsamp=500):
+    """
+    Sample an observed dataset to reduce the number of data points.
+    Useful for generating example data files that run quickly.
+    E.g., for a BEAST example.
+
+    Parameters
+    ----------
+    filename : str
+        name of the file with the file dataset
+
+    nsamp : int
+        number of samples to keep in new dataset
+    """
+    fulldata = Table.read(filename)
+
+    rng = default_rng(1234)
+    indxs = rng.integers(len(fulldata), size=nsamp)
```

Comment:
Thanks for including the code that does the selection!

Thinking about everything that @cmurray-astro went through to generate a new PHAT example, I'm wondering if we should be including anything about source density.  In particular, do we want to be able to use metal-small for examples that include source density info, or should we save that for the new phat-small?

Reply:
New phat_small example.  The metal_small is definitely motivated by the regression tests.  Best to keep it simple!